.github/workflows/ci.yml

```diff
@@ -13,30 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.9']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.9']
 
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - uses: actions/cache@v4
-      with:
-        path: ~/.cache/pip
-        key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ hashFiles('.github/workflows/ci.yml') }}
-        restore-keys: ${{ runner.os }}-python-${{ matrix.python-version }}-pip
+        cache: 'pip'
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
         python -m pip install tox tox-gh-actions
         sudo apt-get install libmemcached-dev
     - name: Lint
-      if: matrix.python-version == '3.10'
+      if: matrix.python-version == '3.13'
       run: |
         tox -e lint,mypy
     - name: Docs
-      if: matrix.python-version == '3.10'
+      if: matrix.python-version == '3.13'
```

Comment:
This will require changes to `docs-requirements.txt` because `imghdr` was one of the dead batteries modules removed from the Standard Library in Python 3.13.
* #608